frontend/src/views/search/SearchStatistics/CodeStats.tsx

```diff
@@ -42,6 +42,8 @@ function CodeStatsWithoutData(props: CodeStatsProps) {
   const sortStatsByGlobal = useAppSelector((state) => state.settings.search.sortStatsByGlobal);
 
   // global server state (react-query)
+  // TODO does it make sense to only show code stats for the current user here?
+  // I think keyword and tag stats show counts for all users
```

Comment:
yes! Keyword, System User code, and Tag stats should be there for all users

Reply:
Removed the filtering by user ids.